### DISCUSSION TITLE
Is posit.co made in quarto?

### DISCUSSION THREAD

**Discussion by aito123** [quarto-dev/quarto-web#599] [Q&A]
I was just wondering if [posit](https://posit.co/) was made in quarto project web as [quarto](https://quarto.org/). If thats the case, can we take a look to the code? The web has some amazing features that I haven't seen in quarto webpage.

Regards

**Comment by jjallaire** [comment]
https://quarto.org is made in quarto (as you probably already knew) but https://posit.co is not.
